aten/src/ATen/native/TensorTransformations.cpp

```diff
@@ -61,15 +61,30 @@ Tensor flip_cpu(const Tensor& self, IntArrayRef dims) {
     }
   }
 
-  AT_DISPATCH_ALL_TYPES_AND_COMPLEX_AND(at::ScalarType::Bool, in_tensor.scalar_type(), "flip_cpu", [&] {
-    flip_cpu_kernel<scalar_t>(
-      total_dims,
-      stride_contiguous_v,
-      flip_dims_b,
-      in_tensor,
-      out_tensor
-    );
-  });
+  if (in_tensor.is_quantized()) {
```

Comment:
if this pattern becomes common, then we probably want to create a new macro that include both AT_DISPATCH_ALL_TYPES_AND_COMPLEX_AND and AT_DISPATCH_QINT_AND_SUB_BYTE_TYPES

Reply:
I agree. Alternatively, I can copy-paste this code into the quantized folder, and register the quantized kernel there.